src/config.c

```diff
@@ -197,6 +197,7 @@ config_property_definition _generalDefinitions[] = {
 	{ offsetof(general_configuration, upper_case_banners),				"upper_case_banners",			CONFIG_VALUE_TYPE_BOOLEAN,		false,							NULL					},
 	{ offsetof(general_configuration, allow_loading_with_incorrect_checksum),"allow_loading_with_incorrect_checksum",	CONFIG_VALUE_TYPE_BOOLEAN,		false,			NULL					},
 	{ offsetof(general_configuration, steam_overlay_pause),				"steam_overlay_pause",			CONFIG_VALUE_TYPE_BOOLEAN,		true,							NULL					},
+	{ offsetof(general_configuration, scale),							"scale",						CONFIG_VALUE_TYPE_FLOAT,		1.0,							NULL					},
```

Comment:
The default value for floats has to be specified by doing ```{ .value_float = 1.0f }```.
Compiler for some reason is casting it to int otherwise.